plugins/blockvault_client/include/eosio/blockvault_client/connect_mgr_config.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef __CONNECT_MANAGER_CONFIG_H 
+#define __CONNECT_MANAGER_CONFIG_H 
+
+#define CONNECT_MGR_THREAD_SAFE
+#define LOAD_BALANCE
+#define LOAD_BALANCE_RANDOM
+#define CLUSTER_INIT_FILE_PATH        "~/blockvault_cluster.ini"
+#define ALIGN_BYTES          8
+
+#define ASYNC_SEND_QUEUE_SIZE          1024
+#define ASYNC_RECEIVE_QUEUE_SIZE     1024
+
+// {FULL_CONNECTION | RE_CONNECTION}  only define one of them
+// FULL_CONNECTION hold all connection to cluster follower, and also one the connection to leader.   n + 1
+// RE_CONNECTION hold one connection to cluster follower, and one connection to leader.                 1 + 1
+//#define FULL_CONNECTION
+#define RE_CONNECTION
```

Comment:
Lots of defines right here. Perhaps look at how the program `nodeos` handles config stuff:
https://github.com/EOSIO/eos/blob/master/programs/nodeos/CMakeLists.txt#L30
https://github.com/EOSIO/eos/blob/master/programs/nodeos/config.hpp.in

Reply:
Good. use config.hpp.in would be more like a part of eos.